README.md

```diff
@@ -66,7 +74,7 @@ $ conda config --add channels conda-forge
 # To make the installation faster, we recommend using mamba
 $ conda install mamba -c conda-forge
 $ mamba create -n MAESTRO maestro=1.2.1.9999 -c liulab-dfci
```

Comment:
need to change it to 1.2.2

Reply:
@baigal628 can you fix this and do another PR?

Reply:
I think @baigal628 can just push updates to her [baigal628:master](https://github.com/baigal628/MAESTRO). No need for another PR.

Reply:
Sure. What I previously did is that I commit to my fork baigal628:master and PR to this repo. I only tested the LISA local mode. To run in web mode, should we just download the DE gene list and upload it to the LISA website?

Reply:
@baigal628 I think the previous design is that the MAESTRO pipeline will 'connect to LISA website API' and retrieve the result. I don't know if it works now.

Sure. You can commit to your forked baigal628:master and we will see the updates in this PR as long as this PR is not closed.